README.md

```diff
@@ -3,10 +3,16 @@
 [![Coverage Status](https://coveralls.io/repos/github/zalando-incubator/mate/badge.svg?branch=master)](https://coveralls.io/github/zalando-incubator/mate?branch=master)
 [![Go Report Card](https://goreportcard.com/badge/github.com/zalando-incubator/mate)](https://goreportcard.com/report/github.com/zalando-incubator/mate)
 
-Mate synchronizes AWS Route53 or Google CloudDNS records with exposed Kubernetes services and ingresses.
+# !!! Update !!! 
```

Comment:
I'd go with "Status" or something similar. "Update" assumes that someone is following us which is not a valid assumption. I know it's nitpicking but it's all READMEs are about :P